packages/foundry/src/create.ts

```diff
@@ -7,11 +7,21 @@ const BaseRepositoryAddress = "workleap/wl-foundry-cli/templates";
 
 type TemplateId = "host-application" | "remote-module" | "static-module";
 
+const validNpmPackageNameRegex = /^(?:@[a-z0-9-*~][a-z0-9-*._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/;
```

Comment:
Constants should start with an uppercase caracter.

Reply:
I thought the plan was to only validate inputs in the client (create-workleap) project?